overview/default.nix

```diff
@@ -171,8 +169,12 @@ let
             ${metadata.summary}
           </p>
         '')
-        + (optionalString (metadata ? subgrants) ''
-          ${render.subgrants.many metadata.subgrants}
+        + (optionalString (metadata ? subgrants && metadata.subgrants != [ ]) ''
+          <p>
+            This project is funded through these NGI grants:
```

Comment:
Just one small thing, I think this should be:

```suggestion
            This project is funded through these NGI subgrants:
```

Reply:
Is subgrant really a name that Nlnet uses? I thought that was just our terminology.

Reply:
I think they call them `projects`, but our definition of what projects are are different. Calling them `grants` in this case might be a bit confusing since that's already used to refer to Review, Entrust, ... in the project pages. Example:
![image](https://github.com/user-attachments/assets/256c6483-a9cd-41c9-bf75-621d94cd19d4)

IMO we should stick to one terminology, and preferably ours.

Reply:
Makes sense :+1: